src/app/pages/CpsMap/index.test.jsx

```diff
@@ -57,17 +90,159 @@ jest.mock('../../containers/PageHandlers/withData', () => Component => {
   return DataContainer;
 });
 
-jest.mock('../../containers/CpsAssetPageMain', () => {
-  const CpsAssetPageMain = () => <div>CpsAssetPageMain</div>;
+jest.mock('../../containers/PageHandlers/withContexts', () => Component => {
+  const ContextsContainer = props => (
+    <div id="ContextsContainer">
+      <Component {...props} />
+    </div>
+  );
 
-  return CpsAssetPageMain;
+  return ContextsContainer;
 });
 
+const escapedText = text => {
+  const textReplacements = {
+    '&quot;': '"',
+    '&amp;': '&',
+    '&lt;': '<',
+    '&gt;': '>',
+  };
+
+  const replacementsRegex = new RegExp(
+    Object.keys(textReplacements).join('|'),
+    'gi',
+  );
+
+  return text.replace(replacementsRegex, match => textReplacements[match]);
+};
```

Comment:
@jamesdonoh I think this helps transforms text like [this](https://github.com/bbc/simorgh/blob/latest/data/pidgin/cpsAssets/23248703.json#L38) to optimo format (like  what the preprocessor does) just to assert that the transformed text is present on the page. 

I don't have aa detailed context about most of this tests as they were already present in the removed [CpsAssetPageMain tests](https://github.com/bbc/simorgh/blob/latest/src/app/containers/CpsAssetPageMain/index.test.jsx#L57-L70) as part of the MAP implementation, I just moved them over and ensured they were passing like before. cc @ryanmccombe

Reply:
As discussed on the Zoom call, I misunderstood here, I thought they were new tests/logic rather than tests that tests that have been relocated. Happy to leave this as-is unless anyone in the media team has any different views.